src/modules/bootloader/bootloader.conf

```diff
@@ -9,19 +9,19 @@
 # Possible options are 'grub', 'sb-shim' and 'systemd-boot'.
 efiBootLoader: "grub"
 
-# systemd-boot configuration files settings, set kernel and initramfs file names
+# systemd-boot configuration files settings, set kernel search path, kernel name
 # and amount of time before default selection boots
-kernel: "/vmlinuz-linux"
-img: "/initramfs-linux.img"
-fallback: "/initramfs-linux-fallback.img"
+kernelSearchPath: "/usr/lib/modules"
+kernelName: "vmlinuz"
 timeout: "10"
 
-# Optionally set the menu entry name and kernel name to use in systemd-boot.
+# additionalInitrdFiles is a comma seperated list of file names
+additionalInitrdFiles: "/boot/amd-ucode,/boot/intel-ucode"
```

Comment:
If this is supposed to be a list? Why not make it a list in YAML then? Notation is with bullets or `[ ]`, and then you get a list in the job-configuration, rather than a string that you need to split and handle separately.